docs/contributing/Local_Development.mdx

```diff
@@ -86,7 +86,16 @@ You can run `yarn typecheck` in any package or in the root to run `tsc`.
 ## Website Development
 
 Our interactive documentation website is built with [Docusaurus](https://docusaurus.io).
-Running `yarn start` from either the root or `packages/website` will start a local dev server on `localhost:3000`.
-
-> The `website` package relies on other packages being built.
-> We recommend running `yarn build` from the root before `yarn start`.
+After cloning the repository, run the following steps to install and build the website:
+
+```sh
+yarn install # To ensure that the dependencies are installed.
+yarn build # This builds all the packages except for `website-eslint` and `website`.
```

Comment:
```suggestion
Before running `yarn start` you have to execute the following commands to build the necessary packages:

```sh
```

Reply:
In theory, you can't skip the setup.

Reply:
In this case, we should just say "remember to run the setup steps at the top of the page".